internal/flow/release.go

```diff
@@ -24,8 +24,65 @@ import (
 func (s *Service) ReleaseBranch(ctx context.Context, actor Actor, environment, service, branch string) (string, error) {
 	span, ctx := s.Tracer.FromCtx(ctx, "flow.ReleaseBranch")
 	defer span.Finish()
-	var result string
+	sourceConfigRepoPath, close, err := git.TempDirAsync(ctx, s.Tracer, "k8s-config-release-branch")
```

Comment:
Why so much new code? Is it moved from somewhere?

Reply:
With indentions removed it made a lot more sense. Thanks for explaining 👍

Reply:
We found out that showing the diff without whitespace changes fixed that 🎉